homeassistant/components/zwave_js/__init__.py

```diff
@@ -178,6 +178,11 @@ async def async_on_node_ready(node: ZwaveNode) -> None:
             if disc_info.assumed_state:
                 value_updates_disc_info.append(disc_info)
 
+        # Create a node status sensor for each device
+        async_dispatcher_send(
```

Comment:
This will only work after at least another sensor has been added for the config entry. I think we need to make it more robust.

Reply:
good catch